Makefile.include

```diff
@@ -1,8 +1,15 @@
 INSTALL=/usr/bin/install
 OBS_BACKEND_PREFIX=/usr/lib/obs/server
+OBS_BACKEND_DATA_DIR=/srv/obs
+OBS_BACKEND_LOG_DIR=$(OBS_BACKEND_DATA_DIR)/log
+OBS_BACKEND_SERVICE_LOG_DIR=$(OBS_BACKEND_DATA_DIR)/service/log
 OBS_DOCUMENT_ROOT=/srv/www/obs
 OBS_API_PREFIX=$(OBS_DOCUMENT_ROOT)/api
+OBS_API_LOG_DIR=$(OBS_API_PREFIX)/log
```

Comment:
Introducing path variables to make the installation more consistent makes sense to me.